dynamic_forms/forms.py

```diff
@@ -264,8 +264,22 @@ def get_fields(properties):
                     widget_class = field_class.widget
 
                 # copy direct options
-                field_args = {k: prop[l] for l, k in cls.ARG_MAP.items() if l in prop}
-                widget_attrs = {k: prop[l] for l, k in cls.WIDGET_ATTR_MAP.items() if l in prop}
+                field_args = {}
+                for l, k in cls.ARG_MAP.items():
+                    if l in prop:
+                        if l in ['title', 'description']:
+                            field_args[k] = translate_lazy(prop[l], i18n)
+                        else:
+                            field_args[k] = prop[l]
+                # field_args = {k: prop[l] for l, k in cls.ARG_MAP.items() if l in prop}
```

Comment:
I guess something like this is the simplest and cleanest:
```
ARGS_WITH_TRANSLATIONS = ('label', 'help_text')
```

```
for arg in ARGS_WITH_TRANSLATIONS:
  if arg in field_args:
    field_arg[arg] = translate_lazy(field_arg[arg], i18n)
```
And similar for the widget attr. Note that, in this way we need to use django attribute names.